data/data/aws/route53/variables.tf

```diff
@@ -14,6 +14,12 @@ variable "etcd_ip_addresses" {
   default     = []
 }
 
+variable "etcd_ipv6_addresses" {
+  description = "List of string IPs (IPv6) for machines running etcd members."
+  type        = list(string)
+  default     = []
```

Comment:
is the default required? if yes can you document what happens when empty vs non-zero?

Reply:
This is exactly like `etcd_ip_addresses` above

Reply:
also similar question to https://github.com/openshift/installer/pull/2555#discussion_r354966038

Reply:
Sure, I can combine this one.